packages/fast-storybook-presets/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@microsoft/fast-storybook-presets",
+  "version": "1.0.0",
+  "description": "Storybook presets for FAST integration",
+  "main": "dist/index.js",
+  "author": {
+    "name": "Microsoft",
+    "url": "https://discord.gg/FcSNfg4"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Microsoft/fast-dna.git"
+  },
+  "bugs": {
+    "url": "https://github.com/Microsoft/fast-dna/issues/new/choose"
+  },
+  "scripts": {
+    "prepare": "tsc -p ./tsconfig.json",
+    "prettier": "prettier --config ../../.prettierrc --write \"**/*.{ts,tsx}\"",
+    "prettier:diff": "prettier --config ../../.prettierrc \"**/*.{ts,tsx}\" --list-different",
+    "test": "npm run tslint",
+    "tslint": "tslint -c ./tslint.json '**/*.ts{,x}'",
+    "tslint:fix": "tslint -c ./tslint.json --fix '**/*.ts{,x}'",
+    "watch": "tsc -p ./tsconfig.json -w --preserveWatchOutput"
+  },
+  "dependencies": {
+    "@microsoft/fast-components-styles-msft": "^4.14.0",
+    "@microsoft/fast-storybook-design-system-addon": "^1.0.0",
+    "@storybook/addon-a11y": "^5.1.9",
+    "@storybook/addon-actions": "^5.1.9",
+    "@storybook/addon-viewport": "^5.1.9",
+    "glob": "^7.1.4",
+    "lodash": "^4.17.11"
```

Comment:
lodash-es?

Reply:
I'm using it in `nodejs` and it was blowing up cause node doesn't understand ES6. Since this is a dev dep only I think this is okay